e2e/support/helpers/e2e-ui-elements-overflow-helpers.js

```diff
@@ -34,12 +34,16 @@ export const assertDescendantNotOverflowsContainer = (
     return;
   }
 
-  expect(descendantRect.bottom, `${message} bottom`).to.be.lte(
+  const getMessage = suffix => {
+    return [message, suffix].filter(Boolean).join(" ");
```

Comment:
Handling the case when `message` is `undefined`

Reply:
Can we use `isNotNull` for this?

Reply:
It wouldn't handle empty strings as expected.